code/http4s/src/main/scala/com/avast/datadog4s/extension/http4s/MetricsOpsBuilder.scala

```diff
@@ -0,0 +1,48 @@
+package com.avast.datadog4s.extension.http4s
+
+import cats.effect.Sync
+import cats.effect.concurrent.Ref
+import cats.syntax.functor._
+import com.avast.datadog4s.api.{ MetricFactory, Tag }
+import com.avast.datadog4s.extension.http4s.DatadogMetricsOps.ClassifierTags
+import com.avast.datadog4s.extension.http4s.impl.{ ActiveConnections, DefaultMetricsOps }
+import org.http4s.metrics.MetricsOps
+
+final case class MetricsOpsBuilder[F[_]: Sync] private (
```

Comment:
nitpick: using the case class is no optimal as it exposes `copy` method. But I won't block you on this.